src/gui/tray/Window.qml

```diff
@@ -373,6 +427,13 @@ Window {
                     visible: UserModel.currentUser.hasLocalFolder
                     icon.source: "qrc:///client/theme/white/folder.svg"
                     onClicked: UserModel.openCurrentAccountLocalFolder()
+
+                    Accessible.role: Accessible.Button
+                    Accessible.name: qsTr("Local folder button")
+                    Accessible.description: qsTr("Opens the local folder corresponding to the currently selected account")

```

Comment:
Here the Accessible.name should be "Open local folder of current account" and then no description is needed.